pkg/config/config.go

```diff
@@ -112,6 +120,46 @@ func (c *ConfigInfo) writeToFile() error {
 	return nil
 }
 
+// SetConfiguration Set Odo configurations
+func SetConfiguration(component string, value bool) error {
+	cfg, err := New()
+	if err != nil {
+		return errors.Wrap(err, "unable to modify configuration")
+	}
+
+	cfg.OdoConfigurations.UpdateNotification = value
+	err = cfg.writeToFile()
+	if err != nil {
+		return errors.Wrapf(err, "unable to set updatenotification to %t ", value)
+	}
+	return nil
+}
+
+// Getconfiguration get odo configurations
+func GetConfiguration() error {
+	cfg, err := New()
```

Comment:
You shouldn't call `New()` here. Check other function in this file, and try to follow the same pattern in your functions.